design/sketch.webidl

```diff
@@ -105,16 +105,51 @@ interface WebGPUTexture {
     WebGPUTextureView createTextureView(WebGPUTextureViewDescriptor desc);
 };
 
-// Sampler
+// Samplers
+enum WebGPUAddressMode {
+    "clampToEdge",
+    "mirrorClampToEdge",
```

Comment:
note: this requires "VK_KHR_sampler_mirror_clamp_to_edge". It's not supported on Mali/Adreno Android GPUs, and a few desktop ones (e.g. Kabylake GT2 on Linux). Perhaps we can get started without it?